getting_started.md

```diff
@@ -0,0 +1,169 @@
+
+```bash
+trident list --remote --packages
+```
+
+You will see a long list of packages. Let's check which packages has French individuals
+
+```bash
+trident list --remote --groups --raw | grep 'French'
+```
+
+Among the packages listed you should find `2012_PattersonGenetics`. 
+Let's download that one, and let's also add three further packages. We will download them into a new directory named `~/poseidon_repo` right in your home folder:
+
+?> Tipp: You can also use `trident list` to view entities in your own local repository. You can list packages, groups or individuals. Check out `trident list --help` or the [documentation](trident.md)
+
+```bash
+mkdir -p ~/poseidon_repo
+trident fetch -f '*2012_PattersonGenetics*,*Reference_Genomes*,*Archaic_Humans*' -d ~/poseidon_repo
+```
+
+## Simple analyses
+
+Let's analyse the newly downloaded data using some F-statistics. Let's show that people with West-Eurasian ancestry are significantly more closely related to Neanderthals than people with West-African ancestry are:
+
+```bash
+xerxes fstats -d ~/poseidon_repo --stat 'F4(<Chimp.REF>,<Altai_published.DG>,Yoruba,French)' \
+  --stat 'F4(<Chimp.REF>,<Altai_published.DG>,Sardinian,French)'
+```
+
+Here is the result:
+```
+.-------------------------------------------------------.-----------------------.-----------------------.--------------------.
+|                       Statistic                       |       Estimate        |        StdErr         |      Z score       |
+:=======================================================:=======================:=======================:====================:
+| F4(<Chimp.REF>,<Altai_published.DG>,Yoruba,French)    | 1.5829336247989434e-3 | 1.9812755791513955e-4 | 7.989467197071763  |
+| F4(<Chimp.REF>,<Altai_published.DG>,Sardinian,French) | 7.610136296417463e-5  | 6.920533028304284e-5  | 1.0996459760097628 |
+'-------------------------------------------------------'-----------------------'-----------------------'--------------------'
+```
+
+You can see that the Z-Score for French being more closely related to the Altai Neanderthal than Yoruba are (an ethnic group from Nigeria) is much larger than 3, suggesting high significance, while the Z-score for either Sardinians or French being more closely related to the Altai Neanderthal is not-significant.
+
+?> Tipp: There are many other statistics that you can compute, such as F2, F3 or FST. Check out `xerxes fstats --help` or the [documentation](xerxes.md)
+
+## Creating your own packages
+
+We now assume that you have genotype data available from your own set of ancient or modern samples. We can simulate this situation by downloading an existing package:
+
+```bash
+mkdir -p ~/tmp
+trident fetch -f '*2014_SkoglundScience*' -d ~/tmp
+```
+
+We now forget about the fact that this is already a Poseidon package and just use its genotype data to create a new package:
+
+```bash
+trident init --genoFile ~/tmp/2014_SkoglundScience/2014_SkoglundScience.bed \
```

Comment:
We should certainly use the new interface suggested in https://github.com/poseidon-framework/poseidon-hs/pull/172 here.

Reply:
Yes I agree.